src/app/components/change-email/change-email.component.html

```diff
@@ -0,0 +1,61 @@
+<mat-drawer-container class="example-container">
+  <mat-drawer mode="side" opened>
+    <mat-nav-list>
+        <a mat-list-item routerLink="/settings"> Overview </a>
+        <a mat-list-item> Change Email </a>
+        <a *ngIf="auth.getAccess() < 3" mat-list-item routerLink="/settings/request"> Request Elevated Access </a>
+        <a *ngIf="auth.hasAccessLevel(3)" mat-list-item routerLink="/settings/manage"> Manage User Access Levels </a>
+    </mat-nav-list>
+  </mat-drawer>
+  <mat-drawer-content>
+<div class="card-container">
+  <mat-card class="example-card">
+    <mat-card-header>
+      <mat-card-title>Change Email</mat-card-title>
+    </mat-card-header>
+    <mat-card-content>
+      <form class="example-form" [formGroup]="createEmailForm">
+        <table class="w-100" cellspacing="0">
+            <tr>
+                <td>
+                  <mat-form-field class="w-100"
+                    [ngClass]="submitted && createEmailForm.controls.currentEmail.errors ? 'mat-form-field-invalid' : ''">
+                    <input matInput placeholder="Current Email" formControlName="currentEmail" name="currentEmail" type="email"
```

Comment:
I question whether this has good usability as it looks as if it can be edited like the other fields, but it cannot

Reply:
<img width="409" alt="Screen Shot 2019-07-11 at 2 52 16 PM" src="https://user-images.githubusercontent.com/8813361/61077007-84132500-a3eb-11e9-827d-d41b42608550.png">
I figured out how to disable it.